### ISSUE TITLE
Accessibility issues in Roadmap

### ISSUE THREAD

**Issue by gjacob24** [DMPRoadmap/roadmap#3452]
Please add all accessibility related issues to this issue.

Following are accessibility issues in the roadmap codebase gathered from feedback that was provided from the UCOP electronic accessibility review:
- [ ] The 'email' and 'website' links do not have a clear context for screen readers. Update the link text to be '[organisation] email' and '[organisation homepage]'. Where the organisation name is retrieved from the branding.yml file. (#1452)
- [ ] The Google maps iframe needs a title that describes what it is showing the user. (#1452)
- [ ] Our PDF files have the following accessibility issues (#1453): 
```
1. Missing descriptive titles
2. Do not follow the proper <h1><h2><h3> hierarchy (e.g. start with <h2> just because the font size is preferred)

For information on descriptive PDF titles: https://www.w3.org/TR/WCAG20-TECHS/PDF18.html
```
